### ISSUE TITLE
Curly braces in HTML files getting weird formatting

### ISSUE THREAD

**Issue by trainiac** [prettier/prettier-vscode#719]
In a Vue SSR application you can interpolate raw strings likes this
```html
<!DOCTYPE html>
<html>
  <head>
    {{{ head }}}
    {{{ renderResourceHints() }}}
    {{{ renderStyles() }}}
  </head>
  <body>
    {{{ renderState() }}}
    {{{ renderScripts() }}}
  </body>
</html>
```
When running prettier via command line or in the [prettier playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAhAEQPIGEAqAmgAoCiABABYwC2ANgHwA6Uq19zU55bcAhgBNO3bsDFV+A8gF9ZLEeTHByAJwQC4KgEpwAzhACuKsHAASAS1i6AFAEoZcriKWr1mgMowAnnT12H0vI8APSUksI8AEYQAl4RouJqUBoqnnzw-rKBTgnKSSnuYCrmAA4wNvZZQajB0bGcNeyMIAA0IBBl5tC6yKB8KioQAO7E-Qg9KHwAbhDmAq0gkSp8YADWcDDuJSuWAObIMCoGcG1NAOqU5vC62ybu41fmU1deyOC6PW2WupowxMu7Gh8ZAAMz4dB+bQAVroAB4AIWWaw27j4NDgABlLHBQeDISAYbD3HtfABFAwQeC4iEnEDbFQ-FRvJoLErFWBnOYwSjIAAcAAY2myID8zssSm82XpNFMcW01ABHAzmNT-PiA4FIME0to-Gjman43QkuDkyk4pCHY5tGB8SKcgTc5AAJhty3MdD2uAgNCBbz0AFYFgYfvg7RNtT9ZEA) you get the expected result .
```html
<!DOCTYPE html>
<html>
  <head>
    {{{ head }}} {{{ renderResourceHints() }}} {{{ renderStyles() }}}
  </head>
  <body>
    {{{ renderState() }}} {{{ renderScripts() }}}
  </body>
</html>
```

However when running pretter via VS Code the output is different
```html
<!DOCTYPE html>
<html>
  <head>
    {{{ head 
    }}} 
    {{{ renderResourceHints() 
    }}} 
    {{{ renderStyles() 
    }}}
  </head>
  <body>
    {{{ renderState() }}} {{{ renderScripts() }}}
  </body>
</html>
```
It's weird it's only broken in the head tag and every time you save it adds an additional space before the ending `}}}`

**Comment by trainiac** [comment]
This is my `settings.json` file
```json
    // Place your settings in this file to overwrite the default settings
    {
        "[javascript]": {
            "editor.formatOnSave": true
        },
        "[vue]": {
            "editor.formatOnSave": true
        },
        "[scss]": {
            "editor.formatOnSave": true
        },
        "[html]": {
            "editor.formatOnSave": true
        },
        "css.validate": false,
        "editor.minimap.enabled": false,
        "editor.formatOnSave": false,
        "editor.fontSize": 14,
        "editor.tabSize": 2,
        "eslint.validate": [
            "javascript",
            "vue",
            "html"
        ],
        "files.trimTrailingWhitespace": true,
        "html.validate.scripts": false,
        "html.validate.styles": false,
        "javascript.validate.enable": false,
        "prettier.disableLanguages": [],
        "scss.validate": false,
        "stylelint.enable": true,
        "vetur.completion.autoImport": false,
        "vetur.format.defaultFormatter.html": "none",
        "vetur.format.defaultFormatter.css": "none",
        "vetur.format.defaultFormatter.postcss": "none",
        "vetur.format.defaultFormatter.scss": "none",
        "vetur.format.defaultFormatter.js": "none",
        "vetur.validation.script": false,
        "vetur.validation.template": false,
        "vetur.validation.style": false,
        "window.zoomLevel": 1,
    }
```

Other installed extensions are:
- Babel ES6/ES7 for syntax highlighting
- ESLint
- SCSS Intellisense
- stylelint
- Vetur

I don't think any of those would be causing the issue.

**Comment by shreyas-a** [comment]
Even I'm experiencing this issue, I tried formatting using prettier CLI & it works fine.

**Comment by ntotten** [comment]
This is fixed in the preview of version 3.0. You can test it out here: https://marketplace.visualstudio.com/items?itemName=Prettier.prettier-vscode
